parachain/runtime/litentry/src/lib.rs

```diff
@@ -230,7 +231,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	impl_name: create_runtime_str!("heima"),
 	authoring_version: 1,
 	// same versioning-mechanism as polkadot: use last digit for minor updates
-	spec_version: 9233,
+	spec_version: 9240,
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
 	transaction_version: 1,
```

Comment:
do we need to bump `transaction_version` ?

Reply:
I think it doesn't need to - but safer if any third party relies on it

I'll bump it